### PR TITLE
feat(swr): add resources and examples

### PR DESCRIPTION
**Pull request by gaetanars**
### Description of your changes

I have:

- [x] Run `make reviewable test` to ensure this PR is ready for review.

### How has this code been tested
```
❯ k get managed
NAME                                        READY   SYNCED   EXTERNAL-NAME                      AGE
user.iam.flexibleengine.upbound.io/user-1   True    True     73c2a8be87844016b5a4cdc62d39255d   65m

NAME                                                                            READY   SYNCED   EXTERNAL-NAME   AGE
repositorysharing.swr.flexibleengine.upbound.io/example-swr-repositorysharing   True    True     OCB0005117      3m39s

NAME                                                                            READY   SYNCED   EXTERNAL-NAME              AGE
organizationusers.swr.flexibleengine.upbound.io/example-swr-organizationusers   True    True     example-swr-organization   46m

NAME                                                              READY   SYNCED   EXTERNAL-NAME            AGE
repository.swr.flexibleengine.upbound.io/example-swr-repository   True    True     example-swr-repository   28m

NAME                                                                  READY   SYNCED   EXTERNAL-NAME              AGE
organization.swr.flexibleengine.upbound.io/example-swr-organization   True    True     example-swr-organization   84m
```

### CODE REVIEW

File: examples/iam/user.yaml.extra [hunk 1]
```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
```
Comment (azrod) [inline examples/iam/user.yaml.extra:1]: Please use `user.yaml.secret` for secret file

File: examples/swr/repositorysharing.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  forProvider:
+    deadline: forever
+    organizationSelector:
+      matchLabels:
+        testing.upbound.io/example-name: example_swr_organization
+    permission: pull
+    repositorySelector:
+      matchLabels:
+        testing.upbound.io/example-name: example_swr_repository
+    sharingAccount: OCB000xxxx
```
Comment (azrod) [inline examples/swr/repositorysharing.yaml:19]: isn't it better to use a secret for this ?

File: examples/swr/repository.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  labels:
+    testing.upbound.io/example-name: example_swr_repository
+  name: example-swr-repository
+spec:
+  forProvider:
+    organizationSelector:
+      matchLabels:
+        testing.upbound.io/example-name: example_swr_organization
+    description: Test repository
+    name: 'example-swr-repository'
```
Comment (azrod) [inline examples/swr/repository.yaml:15]: quote is not required

File: examples/swr/repository.yaml [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    isPublic: false
+    category: 'linux'
```
Comment (azrod) [inline examples/swr/repository.yaml:17]: quote is not required

File: examples/swr/organizationusers.yaml [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+spec:
+  forProvider:
+    organizationSelector:
+      matchLabels:
+        testing.upbound.io/example-name: example_swr_organization
+    users:
+      - userIdSelector:
+          matchLabels:
+            testing.upbound.io/example-name: user_1
+        permission: "Read"
```
Comment (azrod) [inline examples/swr/organizationusers.yaml:18]: quote is not required

File: examples/swr/repositorysharing.yaml (hunk 2 above)
Comment (gaetanars) [reply examples/swr/repositorysharing.yaml:19]: Domain Name isn't a secret value.

**Review by azrod — APPROVED** [review]

**Pull request merged**